rootfs/etc/haproxy/template/haproxy.tmpl

```diff
@@ -127,7 +127,7 @@ frontend httpfront
 ###### HTTPS frontend (tcp mode)
 ######
 frontend httpsfront
-    bind *:443
+    bind *:443{{ if $cfg.UseProxyProtocol }} accept-proxy{{ end }}
```

Comment:
I'd like to see the stats page working if the user for any reason want to expose it. Planning to include a proper URL to be used for health-check as described in #21